lib/js_of_ocaml/json.ml

```diff
@@ -20,6 +20,88 @@
 open Js
 open! Import
 
+(****)
```

Comment:
The write logic for basic types seems to be copied from lib/deriving_json, we could write a comment about this so that future us remembers that there are multiple copies of this logic.

Reply:
Done